lib/ansible/executor/module_common.py

```diff
@@ -521,11 +528,13 @@ def _find_snippet_imports(module_name, module_data, module_path, module_args, ta
         module_substyle = 'jsonargs'
     elif b'WANT_JSON' in module_data:
         module_substyle = module_style = 'non_native_want_json'
+    elif _is_binary(module_path):
+        module_substyle = module_style = 'binary'
```

Comment:
We might want this to be the first thing we check since ```if REPLACER in module_data``` is substituting something in the file.